go/oasis-node/cmd/debug/fixgenesis/fixgenesis.go

```diff
@@ -341,6 +341,9 @@ type v4GovernanceConsensusParameters struct {
 	// UpgradeCancelMinEpochDiff is the minimum number of epochs between the current
 	// epoch and the proposed upgrade epoch for the upgrade cancellation proposal to be valid.
 	UpgradeCancelMinEpochDiff beacon.EpochTime `json:"upgrade_cancel_min_epoch_diff,omitempty"`
+
+	// EnableChangeParametersProposal is true iff change parameters proposals are allowed.
+	EnableChangeParametersProposal bool `json:"enable_change_parameters_proposal,omitempty"`
```

Comment:
There should be no need to modify fixgenesis as `v4` refers to consensus layer v4 which did not have this field.